<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InteractionPanel</class>
 <widget class="QWidget" name="InteractionPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="open_graph_btn">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>10</y>
     <width>470</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Open graph</string>
   </property>
  </widget>
  <widget class="QPushButton" name="redraw_btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>840</y>
     <width>470</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    font: 16px &quot;JetBrains Mono&quot;;
    color: rgb(38, 173, 19);
}

QPushButton:hover {
    background-color: green;
    font: 16px &quot;JetBrains Mono&quot;;
    color: black;
    border: 1px solid transparent;
}</string>
   </property>
   <property name="text">
    <string>Redraw</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dfs_btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>DFS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bfs_btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    font: 16px &quot;JetBrains Mono&quot;;
    color: rgb(38, 173, 19);
}

QPushButton:hover {
    background-color: green;
    font: 16px &quot;JetBrains Mono&quot;;
    color: black;
    border: 1px solid transparent;
}</string>
   </property>
   <property name="text">
    <string>BFS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dfs_input">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>83</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Start vertex</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bfs_input">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>154</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Start vertex</string>
   </property>
  </widget>
  <widget class="QPushButton" name="get_shortest_path_between_vertices_btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>270</y>
     <width>470</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Get Shortest Path Between Vertices</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tsm_btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>780</y>
     <width>470</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    font: 16px &quot;JetBrains Mono&quot;;
    color: rgb(38, 173, 19);
}

QPushButton:hover {
    background-color: green;
    font: 16px &quot;JetBrains Mono&quot;;
    color: black;
    border: 1px solid transparent;
}</string>
   </property>
   <property name="text">
    <string>Traveling</string>
   </property>
  </widget>
  <widget class="QPushButton" name="least_spanning_tree_btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>210</y>
     <width>470</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    font: 16px &quot;JetBrains Mono&quot;;
    color: rgb(38, 173, 19);
}

QPushButton:hover {
    background-color: green;
    font: 16px &quot;JetBrains Mono&quot;;
    color: black;
    border: 1px solid transparent;
}</string>
   </property>
   <property name="text">
    <string>Get Least Spanning Tree</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="path_table">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>470</y>
     <width>470</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string>Vertex</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Path</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cost</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="first_vertex_shortest_path_input">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>340</y>
     <width>140</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>   Start vertex</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="end_vertex_shortest_path_input">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>140</width>
     <height>41</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>   End vertex</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="shortest_path_result">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>140</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>   Cost</string>
   </property>
  </widget>
  <widget class="QPushButton" name="get_shortest_path_between_all_vertices_btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>400</y>
     <width>470</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: transparent;
    font: 16px &quot;JetBrains Mono&quot;;
    color: rgb(38, 173, 19);
}

QPushButton:hover {
    background-color: green;
    font: 16px &quot;JetBrains Mono&quot;;
    color: black;
    border: 1px solid transparent;
}</string>
   </property>
   <property name="text">
    <string>Get Shortest Path Between All Vertices</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
